### DISCUSSION TITLE
Skywalking Profiling capability for Node.js

### DISCUSSION THREAD

**Discussion by lstephe** [apache/skywalking#9885] [General]
Skywalking Profiling capability for Node.js  - When will the Skywalking Profile capability be available for Node.js?  Is it available in the latest Skywalking version 9.2?

**Comment by Superskyyy** [comment]
Hello there, Apache SkyWalking is maintained by the community. And there is no formal schedule for releasing functionalities. A functionality (in your case profiling for Node.js) will only be available if someone decide to contribute. 

We welcome you to discuss and work on this.
